src/GraphQL/Reflection/AccessorFieldResolver.cs

```diff
@@ -1,33 +0,0 @@
-using GraphQL.Resolvers;
-
-namespace GraphQL.Reflection
-{
-    internal class AccessorFieldResolver : IFieldResolver
-    {
-        private readonly IAccessor _accessor;
-        private readonly IServiceProvider _serviceProvider;
-
-        public AccessorFieldResolver(IAccessor accessor, IServiceProvider serviceProvider)
-        {
-            _accessor = accessor;
-            _serviceProvider = serviceProvider;
-        }
-
-        public object? Resolve(IResolveFieldContext context)
-        {
-            var arguments = ReflectionHelper.BuildArguments(_accessor.Parameters, context);
```

Comment:
My goal is to delete the BuildArguments method.  So far it is still used elsewhere, for instance by the `FieldDelegate` method.